client/app/css/source.less

```diff
@@ -4,6 +4,7 @@
 @oui-path: "ovh-ui-kit";
 
 // Vendors
+@import (less) "flatpickr/dist/flatpickr.min.css";
```

Comment:
Vendors could probably be imported in `index.js` if you don't need to use variables or mixin in your sources